lib/spack/spack/cmd/make_installer.py

```diff
@@ -97,13 +97,13 @@ def make_installer(parser, args):
                                     "share/spack/logo/favicon.ico")
 
         try:
-            subprocess.check_call(
-                ('"%s" -S "%s" -B "%s" -DSPACK_VERSION=%s '
-                 '-DSPACK_SOURCE="%s" -DSPACK_LICENSE="%s" '
-                 '-DSPACK_LOGO="%s" -DSPACK_GIT_VERBOSITY="%s"')
-                % (cmake_path, source_dir, output_dir, spack_version, spack_source,
-                   spack_license, spack_logo, git_verbosity),
-                shell=True)
+            spack.util.executable.Executable(cmake_path)(
```

Comment:
This change can be ignored for the moment. Seem to have chopped off the commit where I fix this during rebase.